Dockerfile.pr-builder

```diff
@@ -73,6 +73,8 @@ RUN wget -O qctool_v2.0.1-CentOS6.8-x86_64.tgz http://www.well.ox.ac.uk/~gav/res
     rm -rf qctool_v2.0.1-CentOS6.8-x86_64.tgz && \
     ln -s /qctool/qctool /bin/qctool
 
+RUN wget -O /usr/local/include/catch.hpp https://raw.githubusercontent.com/catchorg/Catch2/master/single_include/catch2/catch.hpp
```

Comment:
I think we should pull this from a location with a version.  We can grab from the releases:

https://github.com/catchorg/Catch2/releases

or the tagged versions:

https://github.com/catchorg/Catch2/tree/v2.4.2

Reply:
ok, done. both of those link to the same header file.